modules/apostrophe-assets/README.md

```diff
@@ -104,7 +104,7 @@ cleaning up obsolete static asset bundles in uploadfs. The default
 is 5 minutes. The assumption is that all production servers have received
 the new deployment and finished serving any straggler HTTP requests 5 minutes after
 a new version is first launched.
-See [Deploying Apostrophe in the Cloud with Heroku](https://docs.apostrophecms.org/apostrophe/tutorials/howtos/deploying-apostrophe-in-the-cloud-with-heroku)
+See [Deploying Apostrophe in the Cloud with Heroku](/tutorials/devops/deployment/deploying-apostrophe-in-the-cloud-with-heroku.md)
```

Comment:
Is this the right URL pattern? I wouldn't know not having spent a lot of time with gitbook lately.

Reply:
Both work, but I generally have been linking the specific file with the .md extension except for READMEs which I've switched to just referencing the folder since that looks a little cleaner.